src/app/downloads/downloads-table.component.css

```diff
@@ -54,6 +54,9 @@ table.scroll-x th:first-child, table.scroll-x td:first-child {
 .table-navigation {
   display: flex;
   justify-content: space-between;
+  padding: 3px;
+  background-color: #fff;
```

Comment:
2 votes for remove the background
https://prx.slack.com/archives/G82E05W2G/p1518707335000076

If the buttons stay white, they aren't gonna look like ghost buttons as much, but they aren't btn-link either.